deploy-example/kubernetes-manifest/1_rbac.yaml

```diff
@@ -5,13 +5,39 @@ metadata:
   namespace: imagepullsecret-patcher
 ---
 apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    k8s-app: image-pull-secret-patcher
+  name: image-pull-secret-patcher
```

Comment:
to stay consistent with the project name, please consider change to `imagepullsecret-patcher`

Reply:
Done